modules/serverless-logic-install-kn-workflow-binary-file.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-binary-file_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin using the binary file
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have installed Podman on your local machine. 
+
+.Procedure
+
+. Download the latest binary file, suitable for your environment, from the link:https://mirror.openshift.com/pub/cgw/serverless-logic/latest/[Serverless Logic download mirror] page.
```

Comment:
I suggest we split it per OS, same as KNative parts.

![image](https://github.com/user-attachments/assets/fc9211c1-7126-47cf-9a26-18544a939d0d)